index.html

```diff
@@ -35,184 +32,163 @@
                             <li><a href="#">About</a></li>
                             <li><a href="#">Contact</a></li>
                         </ul>
-                </nav>
-                </label>
-            </div>
-        </div>
+                    </label>
+                </nav>     
     </header>
 
-    <!--Hero Page-->
-
-    <section>
-        <div class="background">
-            <div class="container">
-                <div class="maintext">
-                    <h1>Hey there. I’m <br>Lukman
-                        <span class="h1-highlight"><br>I’m a software <br>developer</span></h1>
-                    <div class="subtext">
-                        <p> I can help you build a product,
-                            feature or website. Look through some of my work
-                            and experience! If you like what you see and
-                            have a project you need coded,
-                            don’t hestiate to contact me.</p>
-                        <ul class="social">
-                            <li><a href="https://twitter.com/LukmanAbdulka18"><img src="images\twitter.svg"
-                                        alt="twitter"></a></li>
-                            <li><a href="#"><img src="images\Linkedin.svg" alt="linkedin"></a></li>
-                            <li><a href="#"><img src="images\medium.svg" alt="medium"></a></li>
-                            <li><a href="https://github.com/lukman155"><img src="images\github.svg" alt="github"></a>
-                            </li>
-                            <li><a href="#"><img src="images\angellist.svg"></a></li>
-                        </ul>
-                    </div>
+    <!--Hero/Intro Page-->
 
+    <main class="background flex-container">
+        <ul class="header-desktop">
+            <li><a href="#">Portfolio</a></li>
+            <li><a href="#">About</a></li>
+            <li><a href="#">Contact</a></li>
+        </ul>
+        <h1 class='h1-top-space'>Hey there. I’m Lukman</h1>
+        <h1 class="h1-highlight">I’m a software developer</h1>
+            
+        <p class="hero-text">I can help you build a product,
+            feature or website. Look through some of my work
+            and experience! If you like what you see and
+            have a project you need coded,
+            don’t hestiate to contact me.</p>
+
+        <ul class="social-icons">
+            <li><a href="https://twitter.com/LukmanAbdulka18"><img src="images\twitter.svg" alt="twitter"></a></li>
+            <li><a href="#"><img src="images\Linkedin.svg" alt="linkedin"></a></li>
+            <li><a href="#"><img src="images\medium.svg" alt="medium"></a></li>
+            <li><a href="https://github.com/lukman155"><img src="images\github.svg" alt="github"></a></li>
+            <li><a href="#"><img src="images\angellist.svg"></a></li>
+        </ul>
+    </main>
```

Comment:
- [ ] kindly add the small button as shown in the Figma design it will help us navigate to the next section in future projects 

![image](https://user-images.githubusercontent.com/89282221/193100415-96129849-ae7a-430b-994c-be0e07ea79a0.png)